soc_045_rw1_women_political_representation/README.md

```diff
@@ -0,0 +1,16 @@
+This file describes the data pre-processing that was done to [Social Institutions and Gender Index](https://stats.oecd.org/Index.aspx?DataSetCode=GIDDB2019#) for [display on Resource Watch](https://bit.ly/3ejWtuc).
+
+The source provided this dataset as a csv file accessed through its [website](https://stats.oecd.org/Index.aspx?DataSetCode=GIDDB2019#). 
+
+Below, we describe the main actions performed to process the csv file:
+1. Convert the dataframe from long to wide form, converting the variables into columns.
+2. Convert the years in the 'Year' column to datetime objects and store them in a new column 'datetime'.
+3. Dropping duplicates so that only one country is showed per row.
```

Comment:
```suggestion
3. Drop rows for data aggregated by region or income group (labeled 'all regions' or 'all income groups') so that only disaggregated data remains and data is not duplicated.
```